app/models/vote.rb

```diff
@@ -0,0 +1,22 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  validates :user, presence: true
+  validates :review, presence: true
+  validates :user_id, uniqueness: {
+    scope: :review_id, message: "User can only vote once per review."
+  }
+
+  def self.user_already_voted(user, review)
+    Vote.where("(user_id = #{user.id} AND review_id = #{review})")
+  end
+
+  def self.destroy_user_vote(user, review)
+    review_id = review.id
+    vote = Vote.where("(user_id = #{user.id} AND review_id = #{review_id})")
```

Comment:
Okay, I thought there was something like this - don't use straight-up string interpolation with `where` statements! [One possible syntax](http://guides.rubyonrails.org/active_record_querying.html#conditions) ... [An even better syntax](http://guides.rubyonrails.org/active_record_querying.html#hash-conditions)